src/effects/useTokenBalances.ts

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react'
+import { Network, TokenBalanceDetails } from 'types'
+import { depositApi } from 'api'
+import { tokenApi } from 'api'
+
+interface UseTokenBalanceResult {
+  balances: TokenBalanceDetails[] | undefined
+  error: boolean
+}
+
+async function _getBalances(): Promise<TokenBalanceDetails[]> {
+  const tokens = tokenApi.getTokens(Network.Rinkeby)
+  const userAddress = '0x1111111111111111111111111111111111111111'
+
+  const balancePromises = tokens.map(async (token, index) => {
+    const tokenAddress = token.address
+    const [exchangeWallet, pendingDeposits, pendingWithdraws] = await Promise.all([
+      depositApi.getBalance(userAddress, tokenAddress),
+      depositApi.getPendingWithdrawAmount(userAddress, tokenAddress),
```

Comment:
`getPendingWithdrawAmount` is repeated. Did you mean `getPendingDepositAmount`?

Reply:
totally, good catch